### PR TITLE
Hide the 'thank you' box on new credits page when manual credits present

### PR DESCRIPTION
**Pull request by ibrechin**

### CODE REVIEW

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

File: mtp_cashbook/templates/cashbook/new_credits.html [hunk 1]
```diff
@@ -222,7 +223,7 @@ <h2 class="heading-large">{% trans 'Have you updated NOMIS?' %}</h2>
                 {% endblocktrans %}
                 <span class="mtp-none-selected">{% trans 'You haven’t selected any to process yet.' %}</span>
                 <span class="mtp-some-selected">
-                  <strong><span class="js-SelectionCount"></span></strong>{% trans 'credits selected for processing in NOMIS' %}.
+                  <strong><span class="js-SelectionCount"></span></strong>{% trans ' credits selected for processing in NOMIS' %}.
```
Comment (ushkarev) [inline mtp_cashbook/templates/cashbook/new_credits.html]: the space ought to be outside the `trans` tag or else it won't be translated correctly

**Pull request merged**